src/Tools/AI Test Toolkit/src/AITTestContext.Codeunit.al

```diff
@@ -195,6 +195,15 @@ codeunit 149044 "AIT Test Context"
         AITTestContextImpl.GetAITTestSuite(AITTestSuite);
     end;
 
+    /// <summary>
+    /// Sets the token consumption for the run. Useful if external calls are made outside of AI toolkit.
```

Comment:
Is run the entire suite run or line run?
From the implementation, it seems to be line run and additive. Being a little more explicit here would be great :)